src/Runner.Worker/JobRunner.cs

```diff
@@ -329,49 +342,7 @@ private async Task<TaskResult> CompleteJobAsync(IJobServer jobServer, IExecution
 
             if (_runnerSettings.DisableUpdate == true)
             {
-                try
-                {
```

Comment:
i moved this to its own method in the bottom of the file, so i can easily diff the different between send job completion to run-service vs. actions-service.